deploy/base/easi.yaml

```diff
@@ -29,7 +29,7 @@ data:
   AWS_REGION: us-west-2
   AWS_SES_SOURCE: '\"EASi Local\" <no-reply-local@info.easi.cms.gov>'
   AWS_SES_SOURCE_ARN: ses-arn
-  AWS_S3_FILE_UPLOAD_BUCKET: easiappfileuploads
+  AWS_S3_FILE_UPLOAD_BUCKET: easi-app-file-uploads
```

Comment:
curious - do we have to change the resource name in AWS? will this create a second bucket?

Reply:
Nope - this was just my goof originally when I created the base easi manifest. In DEV, IMPL, PROD: we're using easi-app-file-uploads, this will just mirror that correctly now locally in k8s :)

Reply:
Nice! Thanks Justin 😄